test/e2e/fixtures.go

```diff
@@ -303,3 +305,76 @@ func (e *EchoSecure) Deploy(ns, name string) func() {
 		require.NoError(e.t, e.client.Delete(context.TODO(), deployment))
 	}
 }
+
+var ContourConfiguration = &contour_api_v1alpha1.ContourConfiguration{
```

Comment:
I'd prefer this to be a function that returns a ContourConfig, can help avoid issues where it's inadvertently modified by one test and pollutes other tests. I know you have DeepCopy's where it's used now, just makes it a little less error-prone going fwd.

Reply:
Don't need to block on this, can do it later too.

Reply:
Yup np.